src/tasks/component-installers/devfile-workspace-operator-installer.ts

```diff
@@ -59,64 +91,120 @@ export class DevWorkspaceTasks {
       createNamespaceTask(this.getNamespace(), {}),
       {
         title: `Create ServiceAccount ${this.devWorkspaceServiceAccount} in namespace ${this.getNamespace()}`,
-        task: async (_ctx: any, task: any) => {
+        task: async (ctx: any, task: any) => {
           const exist = await this.kubeHelper.serviceAccountExist(this.devWorkspaceServiceAccount, this.getNamespace())
           if (exist) {
             task.title = `${task.title}...It already exists.`
           } else {
-            const serviceAccountPath = path.join(this.getTemplatePath(), 'service_account.yaml')
+            const serviceAccountPath = path.join(this.getTemplatePath(ctx), 'devworkspace-controller-serviceaccount.ServiceAccount.yaml')
```

Comment:
devworkspace-controller-serviceaccount.ServiceAccount.yaml - maybe we should move such yaml names to the constant with shorter name. In this case it could `serviceAccountYaml`? And also we can reuse constants.